include/image_undistort/camera_parameters.h

```diff
@@ -198,9 +198,10 @@ class StereoCameraParameters {
   bool valid(const CameraSide& side, const CameraIO& io) const;
 
  private:
-  bool generateRectificationParameters();
+  bool generateRectificationParameters(void);
```

Comment:
why add void? From memory this style of syntax, while valid only makes a difference in old C code

Reply:
no reason, just a habbit. you can discard this change if you want